web/pgadmin/static/js/components/Buttons.jsx

```diff
@@ -115,17 +115,26 @@ const StyledButton = styled(Button)(({theme}) => ({
       minWidth: '30px',
     }
   },
+  '&.Buttons-dangerButton': {
+    backgroundColor: theme.otherVars.errorColor,
+    border: theme.otherVars.borderColor,
+    '&:hover': {
+      backgroundColor: theme.palette.error.dark,
+    },
+  },
 
 }));
 
 
 /* pgAdmin primary button */
 export const PrimaryButton = forwardRef((props, ref)=>{
-  let {children, className, size, noBorder, ...otherProps} = props;
```

Comment:
A primarybutton means a button which uses Primary Color. You should not make it danger by passing props.
You could've easily passed `color="error"` to DefaultButton to make a danger button.